tests/js/server/import/import-setup.js

```diff
@@ -54,6 +54,10 @@
   db._drop("UnitTestsImportCsvTypesNumber");
   db._drop("UnitTestsImportCsvTypesString");
   db._drop("UnitTestsImportCsvTypesPrecedence");
+  db._drop("UnitTestsImportCsvMergeAttributes");
+  db._drop("UnitTestsImportCsvMergeAttributesInvalid");
+  db._drop("UnitTestsImportCsvMergeAttributesInvalid2");
+  db._drop("UnitTestsImportCsvMergeAttributesInvalid3");
```

Comment:
I don't think these 2 are needed, because the import will not run for these. It will fail _before_ actually importing something into these collections, so there is no way to create/drop them here.